packages/lesswrong/server/resolvers/dialogueMessageResolvers.ts

```diff
@@ -0,0 +1,22 @@
+import { handleDialogueHtml } from "../editor/conversionUtils";
+import { cheerioParse } from "../utils/htmlUtil";
+import { defineQuery } from "../utils/serverGraphqlUtil";
+
+const extractLatestDialogueMessages = async (dialogueHtml: string, numMessages: number): Promise<String[]> => {
+  if (numMessages <= 0) return Promise.resolve([])
+  const html = await handleDialogueHtml(dialogueHtml)
```

Comment:
Are you calling `handleDialogueHtml` just to handle the older dialogues which didn't have display names stored in the html with the dialogue messages?

Reply:
I was just cargo culting. Seems unneccessary